common/test/keyboards/start_of_sentence_3621/source/start_of_sentence_3621.kmn

```diff
@@ -0,0 +1,50 @@
+﻿c This keyboard provides a reasonably complete example of
+c start-of-sentence detection, for use in testing #3621.
+store(&VERSION) '10.0'
```

Comment:
Should this require `15.0` now?

Reply:
No, I think it should gracefully degrade because there are no breaking file format changes.